docs/cli.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Getting Started with the CLI
+---
+
+import CLITabs from '@theme/CLITabs';
+import TabItem from '@theme/TabItem';
+
+The [Svix CLI](https://github.com/svix/svix-cli) is one of the fastest ways to interact with our API.
+
+With the Svix CLI you can:
+ - Interact with our API
+ - Validate Webhook payloads
+ - Quickly open our docs
+
+In this tutorial, we will learn how to install, update, and use the Svix CLI.
+
+### 1) Install the Svix CLI
+
+<CLITabs>
+<TabItem value="homebrew">
+
+To install the Svix CLI with [Hombrew](https://homebrew.sh), run:
+
+
+
+```sh
+brew install svix/svix/svix
+```
+</TabItem>
+<TabItem value="scoop">
+
+To install the Svix CLI with [Scoop](https://scoop.sh/), run:
+
+
+```sh
+scoop bucket add svix https://github.com/svix/scoop-svix.git
+```
+
+```sh
+scoop install svix
+```
+
+</TabItem>
+<TabItem value="macos">
+
+To install the Svix CLI on macOS without Homebrew:
+
+1. Download the latest Darwin tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Darwin_x86_64.tar.gz`
+
+2. Optionally, install the binary in a location where you can execute it globally (e.g., /usr/local/bin).
+
+</TabItem>
+<TabItem value="linux">
+
+To install the Svix CLI on Linux without a package manager:
+
+1. Download the latest Linux tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Linux_x86_64.tar.gz`
+
+3. Run the executable: `./svix`
+
+</TabItem>
+<TabItem value="windows">
+
+To install the Svix CLI on Windows without Scoop:
+
+1. Download the latest Windows tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the `svix_X.X.X_Windows_x86_64.tar.gz` file.
+
+3. Run the `.exe` file
+
+</TabItem>
+</CLITabs>
+
+### 2) Login to your Svix Account
```

Comment:
I forgot to mention, but this whole section is confusing. This person doesn't care about `login` yet, they are just learning on the CLI for the first time. It should be easy and straightforward as possible. I'd just mention the env var and not talk about login.

Reply:
I rephrased to `Pair with your Svix Account` and simplified it, do you think it makes more sense now or would you still prefer to just say "set the env variable"